src/test/java/org/folio/des/scheduling/acquisition/AcqBaseExportTaskTriggerTest.java

```diff
@@ -147,10 +147,11 @@ void hourlyScheduleIfScheduledTimeIsNotSet() {
     long diff = TimeUnit.HOURS.convert(diffInMilliseconds, TimeUnit.MILLISECONDS);
     assertEquals(expDiffHours, diff);
 
-    ZonedDateTime nowDateTime = Instant.now().atZone(ZoneId.of("UTC")).truncatedTo(ChronoUnit.DAYS);
-    Instant lastInstant = Instant.ofEpochMilli(actDateHourLate.getTime()).truncatedTo(ChronoUnit.DAYS);
+    ZonedDateTime expDateTime =
+      Instant.now().atZone(ZoneId.of("UTC")).plusHours(expDiffHours).truncatedTo(ChronoUnit.DAYS);
+    Instant lastInstant = Instant.ofEpochMilli(actDate.getTime()).truncatedTo(ChronoUnit.DAYS);
     ZonedDateTime zonedDateTime = ZonedDateTime.ofInstant(lastInstant, ZoneId.of("UTC")).truncatedTo(ChronoUnit.DAYS);
-    assertTrue(nowDateTime.isEqual(zonedDateTime));
+    assertTrue(expDateTime.isEqual(zonedDateTime));
```

Comment:
I am comparing with actDate with currently actual Time + expDiffHours

Reply:
and also changed name nowDateTime to expDateTime ( nowDatetime + expDiffhours ) to be more clear